com.github.wwmm.easyeffects.yml

```diff
@@ -0,0 +1,389 @@
+            sources:
+              - type: archive
+                url: 'https://downloads.sourceforge.net/sourceforge/bs2b/libbs2b-3.1.0.tar.gz'
+                sha256: 6aaafd81aae3898ee40148dd1349aab348db9bfae9767d0e66e0b07ddd4b2528
+              - type: script
+                dest-filename: autogen.sh
+                commands:
+                  - cp -p /usr/share/automake-*/config.{sub,guess} build-aux
+                  - autoreconf -vfi
+              - type: patch
+                path: patch/bs2b/001-fix-automake-dist-lzma.patch
+            post-install:
+              - install -Dm644 -t $FLATPAK_DEST/share/licenses/bs2b COPYING
+            cleanup:
+              - /bin
+
+          - name: speexdsp
+            buildsystem: autotools
+            sources:
+              - type: git
+                url: 'https://gitlab.xiph.org/xiph/speexdsp'
+                tag: 'SpeexDSP-1.2.0'
+                commit: 64cbfa9bca7479a758351aa02bb4abdd76baa9e7
+                # note to future Flatpakers: if it's needed, webrtc-audio-processing has a module in the pulseeffects manifest that could replace this one.
+                # https://github.com/flathub/com.github.wwmm.pulseeffects/issues/44#issuecomment-885999963
+          
+          # Ladspa & LV2 Plugins
+          
+          # manifest module from: https://github.com/flathub/io.mpv.Mpv/blob/73681476e1abde88770804c497f03c4bfea027f6/io.mpv.Mpv.yml#L223
+          - name: rubberband
+            buildsystem: meson
+            cleanup:
+              - /include
+              - /lib/*.a
+              - /lib/pkgconfig
+            sources:
+              - type: archive
+                url: https://breakfastquay.com/files/releases/rubberband-1.9.2.tar.bz2
+                sha256: b3cff5968517141fcf9e1ef6b5a1fdb06a5511f148000609216cf182ff4ab612
+                x-checker-data:
+                  type: html
+                  url: https://www.breakfastquay.com/rubberband/
+                  version-pattern: Rubber Band Library v(\d\.\d+\.?\d*) source
+                  url-template: https://breakfastquay.com/files/releases/rubberband-$version.tar.bz2
+
+          - name: calf
+            build-options:
+              env:
+                - NOCONFIGURE=1
+            rm-configure: true
+            sources:
+              - type: archive
+                url: 'https://calf-studio-gear.org/files/calf-0.90.3.tar.gz'
+                sha256: e32bc23fb68f88729bdef361614fd9ac38627c774c35cd3c4eb611fca887a49f
+            post-install:
+              - install -Dm644 -t $FLATPAK_DEST/share/licenses/calf COPYING
+            cleanup:
+              - /bin
+              - /share/applications
+              - /share/bash-completion
+              - /share/calf
+              - /share/doc
+              - /share/icons
+              - /share/man
+            modules:
+              - "shared-modules/linux-audio/fluidsynth2.json"
+          
+          - name: lsp-plugins
```

Comment:
These are available as flatpak. Can't they just be automatically installed? (I know you can, but the question is whether it works for easyeffects)

Reply:
Interestingly it seems there used to be a [lsp Flatpak for PulseEffects](https://github.com/flathub/com.github.wwmm.pulseeffects.plugin.lsp-plugins), but then it was removed in favour of putting it in the manifest.

I tried testing if the lsp-plugins and zam-plugins installed as Flatpak extensions works, but I wasn't really able to see whether they are working properly. Part of the problem is I'm not very familar with how they are supposed to work... 

I'll try testing it more later, if anyone can I'd appreciate comparing the builds [before](https://github.com/flathub/flathub/pull/2429#issuecomment-888746069) and [with the Flatpak extensions](https://github.com/flathub/flathub/pull/2429#issuecomment-889718300). Maybe I'll look at the Arch package too.